e2e-tests/portal/cypress/support/portal/createBssApplicationReadyForCheck.js

```diff
@@ -0,0 +1,81 @@
+const dashboard = require('../../e2e/pages/dashboard');
+const selectScheme = require('../../e2e/pages/selectScheme');
+const beforeYouStart = require('../../e2e/pages/beforeYouStart');
+const bankDetails = require('../../e2e/pages/bankDetails');
+const contract = require('../../e2e/pages/contract');
+const aboutSupplier = require('../../e2e/pages/contract/about/about-supplier');
+const aboutBuyer = require('../../e2e/pages/contract/about/about-buyer');
+const aboutFinancial = require('../../e2e/pages/contract/about/about-financial');
+const eligibilityCriteria = require('../../e2e/pages/eligibilityCriteria');
+const eligibilityPreview = require('../../e2e/pages/eligibilityPreview');
+const bondDetails = require('../../e2e/pages/bondDetails');
+const bondFinancialDetails = require('../../e2e/pages/bondFinancialDetails');
+const bondFeeDetails = require('../../e2e/pages/bondFeeDetails');
+const contractReadyForReview = require('../../e2e/pages/contract/contract-readyForReview');
+const login = require('./login');
+const MOCK_USERS = require('../../../../e2e-fixtures/index');
+
+const { BANK1_MAKER1 } = MOCK_USERS;
+
+const createBssApplicationReadyForCheck = () => {
+  login(BANK1_MAKER1);
+  dashboard.createNewSubmission().click();
+  selectScheme.bss().click();
+  selectScheme.continue().click();
+  beforeYouStart.true().click();
+  beforeYouStart.submit().click();
+  bankDetails.bankDealId().type('12345678');
+  bankDetails.bankDealName().type('BSS Deal');
+  bankDetails.submit().click();
+  contract.aboutSupplierDetailsLink().click();
+  aboutSupplier.supplierType().select('Exporter');
+  aboutSupplier.supplierCompaniesHouseRegistrationNumber().type('12345678');
+  aboutSupplier.supplierSearchCompaniesHouse().click();
+  aboutSupplier.supplierAddress().country().select('United Kingdom');
+  aboutSupplier.supplierCorrespondenceAddressSame().click();
+  aboutSupplier.smeTypeSmall().click();
+  aboutSupplier.supplyContractDescription().type('Supply Contract Description');
```

Comment:
For all `.type` instances in this command, can we please add `{ delay: 0 }`? E.g:

```js
aboutSupplier.supplyContractDescription().type('Supply Contract Description', { delay: 0 });
```

This minimises the time it takes to enter text as much as possible. When this command is consumed in all the relevant E2E tests, this change will significantly reduce the time it takes to execute 👍 

We need to create a generic `keyboardInput` type cypress command, but this is one for another day.

Reply:
Added `{ delay: 0 }` for `.type` instances.

Reply:
Have updated all instances to use `cy.keyboardInput`.